src/components/results/Results.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactElement } from 'react';
+import Card from '../card/Card';
+import Pagination from '../pagination/Pagination';
+import styles from './Results.module.scss';
+import type { SearchResponseResults, Author } from '../../types/types';
+import type { DataResults } from './Results.types';
+
+function getArticlesCards(articles: SearchResponseResults[], authors: Author[]): ReactElement[] {
+    return articles.map((obj: SearchResponseResults, idx: number) =>
+        <Card
+            key={obj.id} {...obj}
+            author={authors[idx]}
+        />
+    );
+}
+
+export default function Results(props: DataResults): ReactElement {
```

Comment:
`Results` or `Card` is kinda bad naming, be more explicit about it. Those are `ArticlesResults` or `ArticlesList` and `ArticleCard` or `ArticleTile` or `ArticlePreview`